etl/steps/data/garden/migration/2024-11-18/migration_between_regions.py

```diff
@@ -0,0 +1,56 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+from etl.data_helpers import geo
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+REGIONS = [reg for reg in geo.REGIONS.keys() if reg != "European Union (27)"]
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load meadow dataset.
+    ds_migration = paths.load_dataset("migrant_stock")
+    ds_regions = paths.load_dataset("regions")
+    ds_income_groups = paths.load_dataset("income_groups")
+
+    # Read table from meadow dataset.
+    tb = ds_migration["migrant_stock_dest_origin"].reset_index()
+
+    # Aggregate regions (twice, once for destination and once for origin).
```

Comment:
I would define a `frac_allowed_nans_per_year` here in case there is less coverage for some regions/years

Reply:
Good point, thanks!